### PR TITLE
[DB+API]:  [DB]: added labels to labels table [API] added LabelController class and connected classes 

### PR DESCRIPTION
**Pull request by DEz-1996**

### CODE REVIEW

File: src/main/java/application/service/impl/LabelServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+public class LabelServiceImpl implements LabelService {
+    private static final String CANT_FIND_BY_ID = "Can't find label with id: ";
+    private final LabelRepository labelRepository;
+    private final LabelMapper labelMapper;
+
+    @Override
+    public LabelResponseDto create(LabelRequestDto labelRequestDto) {
+        Label label = labelMapper.toEntity(labelRequestDto);
+        Label savedLabel = labelRepository.save(label);
+        return labelMapper.toResponseDto(savedLabel);
```
Comment (RostyslavKuzmych) [inline src/main/java/application/service/impl/LabelServiceImpl.java:27]: return labelMapper.toResponseDto(labelRepository.save(labelMapper.toEntity(labelRequestDto);`

File: src/main/java/application/service/impl/LabelServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                .collect(Collectors.toSet());
+    }
+
+    @Override
+    public LabelResponseDto update(LabelRequestDto labelRequestDto, Long id) {
+        Label label = labelRepository.findById(id).orElseThrow(
+                () -> new EntityNotFoundException(CANT_FIND_BY_ID + id));
+        label.setName(labelRequestDto.getName())
+                .setColor(labelRequestDto.getColor());
+        Label savedLabel = labelRepository.save(label);
```
Comment (RostyslavKuzmych) [inline src/main/java/application/service/impl/LabelServiceImpl.java:43]: You can just simplify that: 
labelMapper.toResponseDto(labelRepository.save(label)

File: src/main/resources/db/changelog/changes/11-insert-labels-labels-table.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  -  changeSet:
+       id: insert-labels-labels-table
```
Comment (RostyslavKuzmych) [inline src/main/resources/db/changelog/changes/11-insert-labels-labels-table.yaml:3]: insert into*

**Review by RostyslavKuzmych — CHANGES_REQUESTED** [review]
Just fix this mistakes

**Review by AgroFix — APPROVED** [review]

**Review by AgroFix — APPROVED** [review]

**Review by RostyslavKuzmych — APPROVED** [review]

**Pull request merged**